app/controllers/api/v1/website_members_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::WebsiteMembersController < Api::V1::ApiController
+
+  def index
+    @members = current_site.website_members
+  end
+
+  def destroy
+    if current_member.is_owner_of?(current_site)
+      @website_member = WebsiteMember.find(params[:id])
+      _not_allowed!("Owner cant be removed.") if @website_member.role == 'owner'
```

Comment:
in teorie asta ar trebuii sa fie in validation si sa verifici daca destroy e valid. Nu sa faci o verificare inainte de destroy. Daca nu e clar intreaba-ma pe slack. Ti-am lasat si un task #73 dar vreau alt PR pt el